bin/node/src/commands/genesis/inputs.rs

```diff
@@ -18,18 +18,9 @@ pub struct GenesisInput {
 #[derive(Debug, Clone, Deserialize, Serialize)]
 #[serde(tag = "type")]
 pub enum AccountInput {
-    BasicWallet(BasicWalletInputs),
     BasicFungibleFaucet(BasicFungibleFaucetInputs),
 }
```

Comment:
Should we remove the possibility entirely like this? i.e. is there zero value in having a wallet account in the genesis file?

(I think yes, but just checking)

Reply:
I was thinking about it and decided that for mainnet we will need to predeploy different accounts and contracts (like DAO for initial funds management), but I don't think that predeployment of simple wallets has any sense for us except testing (but for testing purposes we can deploy wallets in tests).